packages/clerk-js/src/ui/elements/VerificationLinkCard.tsx

```diff
@@ -33,8 +33,8 @@ export const VerificationLinkCard = (props: VerificationLinkCardProps) => {
     <Flow.Part part='emailLinkVerify'>
       <Card.Root>
         <Card.Content>
-          <Card.Alert>{card.error}</Card.Alert>
-          <Header.Root>
+          <Header.Root showLogo>
+            <Card.Alert>{card.error}</Card.Alert>
```

Comment:
Why is the error inside the Header? On all the other screens, the error is outside the Header.Root component. 🤔

Reply:
Good catch, fixing!